tracer/test/test-applications/aspnet/Samples.AspNetMvc5/Samples.AspNetMvc5.csproj

```diff
@@ -123,6 +123,7 @@
     <Content Include="Areas\Datadog\Views\_ViewStart.cshtml" />
     <Content Include="Areas\Datadog\Views\Shared\_Layout.cshtml" />
     <Content Include="Areas\Datadog\Views\DogHouse\Index.cshtml" />
+    <Content Include="libman.json" />
```

Comment:
Not sure this should have been included, but it really doesn't matter as this is a sample :D